### DISCUSSION TITLE
TLS 1.0/1.1 deprecation on Microsoft Artifact Registry (MAR)

### DISCUSSION THREAD

**Discussion by terencet-dev** [microsoft/containerregistry#149] [Announcements]
We are writing to let you know that starting in November 2023, Microsoft Artifact Registry (also known as Microsoft Container Registry or MAR/MCR) will begin enforcing Transport Layer Security (TLS) version 1.2 on all connections to and from our platform.
 
To minimize impact, our team will be rolling out the enforcement starting November 2023 and will be monitoring for connections that are still using TLS 1.0 and TLS 1.1. We appreciate you for being a valued MAR user.

**Comment by CMDRWolfzx12r** [comment]
👴😶🧠 java microsoft🤐. 🔒👴😶PC📀PHOTO'S, FILE'S, VID🎥, MU🎶⛓🛡⚰🔒📡

**Comment by CMDRWolfzx12r** [comment]
PHOTO 1800 - 2022 FILE ⚰👴😶🧠
